pkg/model/const.go

```diff
@@ -291,4 +291,7 @@ const (
 
 	// OktetoDivertDefaultHeaderValue the default divert header value
 	OktetoDivertDefaultHeaderValue = "${OKTETO_NAMESPACE}"
+
+	//OktetoDivertAnnotationTemplate annotation for the okteto mutation webhook to divert a virtual service
+	OktetoDivertAnnotationTemplate = "divert.okteto.com/%s-%s"
```

Comment:
can we move this const to` pkg/constants`? we are trying to reduce `model` package

Reply:
sure, I will update the PR :-)